### PR TITLE
React capstone project - Fruitycee webapp

### PR DESCRIPTION
**Pull request by muskan2532150**
### Home page
- When the page loads, the webapp shows the list of categories that could be filtered out by some parameter.
- Along with the category name, you will display some numeric values per category that come from the API.
- When the user clicks (or taps) on a category item, the application navigates to the details page.

### Details page
- In the details page, the webapp retrieves data from the API to show detailed data in the category.
- When the user clicks on the "Back" button (<), the user navigates to the home page.

#### Testing requirements
- Create unit tests for pure functions in your Redux code.
-  Mock the connection to the Redux Store.

### CODE REVIEW

File: src/test/container.test.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+});
+
+test('renders LI Component', () => {
+  const li = render(
+    <Provider store={store}>
+      <Li nutritions={fruit.nutritions} />
+    </Provider>,
+  );
+  expect(li).toMatchSnapshot();
+});
```
Comment (KayLemba) [inline src/test/container.test.js:58]: When i run your tests they are all failing please fix this. All your test cases are supposed to pass please dont overlook this for it is a very important project requirement .

File: README.md [hunk 2]
```diff
@@ -1 +1,93 @@
...
+# Fruitycee
+
+A web application for a provides data for all kinds of fruit.
+
+# Live Demo
+- Add Allow CORS Extension for the api to work.
+- Please click on Logo in the site to make it work.
+
+[Live Demo](https://muskan2532150.github.io/Fruitycee)
+
```
Comment (KayLemba) [inline README.md:12]: Your project is supposed to be deployed using Netlify or Heroku not github pages. Please deploy your project using netlify or Heroku. Just incase you are unsure on how to deploy a react project using netlify kindly follow this link [here](https://www.freecodecamp.org/news/how-to-deploy-a-react-application-to-netlify-363b8a98a985/)

File: README.md [hunk 3]
```diff
@@ -1 +1,93 @@
...
+```bash
+git@github.com:muskan2532150/Fruitycee.git
+```
+
+3. Open terminal and cd into project and install the dependencies
+
+```bash
+cd Space-Travelers-Hub
+
```
Comment (KayLemba) [inline README.md:46]: Please correct this. This is a metrics app project please kindly add the appropriate information for this project

**Review by KayLemba — CHANGES_REQUESTED** [review]
Hi @muskan2532150    👋
Good job so far working on the project requirements!
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight 🎉
No linter errors. ✔️
GitHub flow is followed. ✔️

Required Changes ♻️
Check the comments under the review in the inline comments.
[REQUIRED] While you did a nice job recording a video talking about your project you did not talk about what feature you are most proud of and you are also required to show a peace of your code. Below is what is required of you when you record a loom video about your project.


1. The student recorded a video presenting the features of the project.
2. The maximum length of the video is 5 minutes.
3. The video contains a clear and objective description of the project in professional 
     English. The topic and purpose of the project are made clear.
4. The video contains a demo of the project, highlighting all the project features.
5. The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

- Kindly record another loom video talking about your project but this time around record it based on the  requirements I have provided for above. 

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/redux/Search/search.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+/* eslint no-param-reassign: "error" */
+import { createSlice } from '@reduxjs/toolkit';
```
[the remark this answers is not in the document]
Comment (muskan2532150) [reply src/redux/Search/search.js:2]: @KayLemba, This Eslint is for redux, and I used redux toolkit where we can assign the state directly (internally it works same as redux) but will produce an immutable state error to remove that, I disabled the eslint.

File: README.md (hunk 2 above)
Comment (muskan2532150) [reply README.md:12]: @KayLemba, In the requirements there is nowhere mention that we can't deploy using Github pages

 [Project Requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md).

Comment (KayLemba) [reply README.md:12]: I'd advise you to make the required changes we code reviewers have guidelines we follow I cannot ask you to make a change outside project requirements. 

Please Have a look.

Comment (muskan2532150) [reply README.md:12]: @KayLemba, I saw the Live version and on line 7 it says we can deploy using Heroku ,Netlily or GitHub Pages

Comment (KayLemba) [reply README.md:12]: Ok then sorry about let me delete the change

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @muskan2532150, 

Great work on making the changes requested by a previous reviewer 👏 🔥
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights 🎉🔥
- Almost complete requested changes

 ## Required Changes ♻️
- [ ] The project is not deployed correctly, the home and detail pages are not showing as expected whereas it runs perfectly locally. Let's fix that
Screen Shot 2022-10-08 at 7 55 05 PM


_Check the comments under the review._

## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by peterrobert — APPROVED** [review]
Hi @muskan2532150, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**